scalableminds/webknossos
# Refactor Volume Annotation Merging

Adds some convenience methods to `MergedVolume` to make its usages DRYer and easier to read. Also don’t silently ignore zip unpacking Failures anymore.

### URL of deployed dev instance (used for testing):
- https://refactorvolumemerging.webknossos.xyz

### Steps to test:
- open volume annotation, brush some, drag in other volume zip, should be merged correctly (be wary of #4937)
- upload multiple volume zips, should be merged correctly (blocked by #4915 )
- create, brush and finish volume tasks, click view on volume project, should merge data correctly in CompoundProject

### Issues:
- fixes #4916 

------
